charter.html

```diff
@@ -0,0 +1,330 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Anti-Fraud Community Group Charter
+    </title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="//www.w3.org/StyleSheets/TR/base">
+    <style>
+      body {
+        max-width: 60em;
+        margin: auto;;
+      }
+      *:target {
+        background-color: yellow;
+      }
+      li {
+        margin-bottom: 9pt;
+      }
+      .note {
+        background-color: yellow;
+        padding: 10px;
+      }
+      .remove {
+        background-color: yellow;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>
+      [DRAFT] Anti-Fraud Community Group Charter
+    </h1>
+    <p>
+      This Charter is work in progress. To submit feedback, please use <a href="https://github.com/antifraudcg/antifraudcg.github.io/issues">https://github.com/antifraudcg/antifraudcg.github.io/issues</a>.
+    </p>
+    <h2 id="goals">
+      Goals
+    </h2>
+    <p>
+      The mission of the Anti Fraud Community Group is to identify and define
+      scenarios involving fraud and unwanted traffic, as well as to incubate
+      and develop web features and APIs to address those scenarios while being
+      mindful of user security, privacy, and accessibility. Fraud and unwanted
+      traffic can include web activity perpetrated by botnets,attackers
+      impersonating users, and other activity that intends to deceive and harm
+      users or compromise web services.
+    </p>
+    <p>
+      The group welcomes participation from anti-fraud service providers,
+      common targets of unwanted traffic, browser vendors, web privacy
```

Comment:
I'm not sure "web privacy advocates" is a common category (although I like the idea!), it might just be "privacy advocates" or "privacy and consumer advocates".

Reply:
👍